src/encoded/static/components/filegallery.js

```diff
@@ -115,20 +101,50 @@ export const compileAnalyses = (experiment, files) => {
                     // annotation and add each to the compiled list. Filter out any not included in
                     // the experiment's files.
                     const assemblyFiles = _.uniq(analysesByAssembly[assembly].reduce((accFiles, analysis) => accFiles.concat(analysis.files), []).filter((file) => fileIds.includes(file)));
-                    const { status } = analysesByAssembly[assembly][0];
+                    // eslint-disable-next-line camelcase
```

Comment:
Oh, I guess there's no choice than to disable that rule here. I can't think of an alternative either.